src/modules/CC1101/CC1101.h

```diff
@@ -730,13 +730,26 @@ class CC1101: public PhysicalLayer {
     */
     int16_t setFrequency(float freq);
 
+     /*!
+      \brief gets carrier frequency. 
+      \returns the freq currently set in Mhz
+    */
+    float getFrequency();
+
     /*!
       \brief Sets bit rate. Allowed values range from 0.025 to 600.0 kbps.
       \param br Bit rate to be set in kbps.
       \returns \ref status_codes
     */
     int16_t setBitRate(float br);
 
+    /*!
+      \brief gets the currently set bit rate.
+      \returns the bit rate in kbps
+    */
+    float getBitRate();
+
+
```

Comment:
Both of these methods fall under https://github.com/jgromes/RadioLib/issues/622, I'd rather solve that more globally for all modules. In the meantime, please remove them and use `this->bitRate` and `this->frequency` in the Rx BW method.

Reply:
Removed